key-vault.tf

```diff
@@ -1,29 +1,29 @@
 module "ccpay-vault" {
-  source = "git@github.com:hmcts/cnp-module-key-vault?ref=azurermv2"
+  source = "git@github.com:hmcts/cnp-module-key-vault?ref=master"
   name = join("-", [var.product, var.env])
   product = var.product
   env = var.env
   tenant_id = var.tenant_id
   object_id = var.jenkins_AAD_objectId
   resource_group_name = azurerm_resource_group.rg.name
   # group id of dcd_reform_dev_azure
-  product_group_object_id = "56679aaa-b343-472a-bb46-58bbbfde9c3d"
+  product_group_name = "dcd_reform_dev_azure"
```

Comment:
don't you want to swap to your team group? not this group, people shouldn't be in this group any more, (the replacement group is 'DTS CFT Developers')

Reply:
Hi @timja - Can we swap to this group **dcd_group_fees&pay_v2**?

Reply:
yes you can change to that, that would be my recommendation, no other changes

Reply:
Thanks @timja. I have updated the group. Can you please approve ?